src/parser/TextSearchQuery.h

```diff
@@ -165,7 +189,19 @@ struct TextIndexScanForWordConfiguration {
   }
 
   friend std::ostream& operator<<(
-      std::ostream& os, const TextIndexScanForWordConfiguration& conf);
+      std::ostream& os, const TextIndexScanForWordConfiguration& conf) {
+    os << "varToBindText_: " << conf.varToBindText_.name()
+       << "; word_: " << conf.word_ << "; matchVar_: "
+       << (conf.matchVar_.has_value() ? conf.matchVar_.value().name()
+                                      : "not set")
+       << "; scoreVar_: "
+       << (conf.scoreVar_.has_value() ? conf.scoreVar_.value().name()
+                                      : "not set")
+       << "; isPrefix_: " << (conf.isPrefix_ ? "true" : "false")
+       << "; variableColumns_: "
+       << (conf.variableColumns_.has_value() ? "is set" : "not set");
+    return os;
+  }
```

Comment:
Please move this back to the `.cpp` file.
Here the readability + compile times are more important that the advantages of the "hidden friend" thingy suggested by Sonarcloud.